.github/workflows/CD.yml

```diff
@@ -629,16 +629,34 @@ jobs:
           src: public.ecr.aws/${{ env.ECR_REPO }}:${{ needs.release-checking.outputs.version }}
           dst: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:${{ needs.release-checking.outputs.version }}
 
+  mirror_adot_operator:
+    runs-on: ubuntu-latest
+    needs: release-latest-image
```

Comment:
Why does this job require the `release-latest-image` job to be successful?

Reply:
Looks like it doesn't, but I've added that it needs `release-checking` to be successful instead as its output is needed to compare the docker version.